dpdd.tex

```diff
@@ -184,25 +185,25 @@ \subsection{Classes of LSST Data Products}
+Finally, LSST Survey Specifications (\S~3.4 of LSST \SRD) prescribe that 90\% of LSST observing time be spent in the so-called ``universal cadence'' mode of surveying the sky. These observations will result in Prompt and Data Release data products discussed above. The remaining 10\% of observing time will be devoted to \textbf{special programs}, designed to obtain improved coverage of interesting regions of observational parameter space. Examples include very deep ($r\sim26$, per exposure) observations, observations with very short revisit times ($\sim$1 minute), and observations of ``special'' regions such as the Ecliptic, Galactic plane, and the Large and Small Magellanic Clouds. The data products for these programs will be generated using the same processing software and hardware and possess the general characteristics of Prompt and Data Release data products, but may be performed on a somewhat different cadence. They will be discussed in \S~\ref{sec:specialProgs}.
 
 
 \section{General Considerations}
 
 Most LSST data products will consist of images and/or catalogs. The catalogs will be stored and offered to the users as \emph{relational databases} which they will be able to query. This approach was shown to work well by prior large surveys, for example the Sloan Digital Sky Survey (SDSS).
 
-Different data products will generally be stored in different databases. For example, Level 1 data products will be stored in a \emph{Level 1 database}. Level 2 ``universal cadence'' products will be stored in a \emph{Level 2 database} database. The products for special programs may be stored in many different databases, depending on the nature of the program.
+Different data products will generally be stored in different databases. For example, Prompt data products will be stored in a \emph{Prompt database}. Data Release ``universal cadence'' products will be stored in a \emph{Data Release database}. The products for special programs may be stored in many different databases, depending on the nature of the program.
```

Comment:
@ktlim may want to comment on whether we are really calling it the "Prompt database".

Reply:
I suggest that, even if we are, we should *also* refer to it as the ”Level 1 Database” here: that's what we're using in internal design documentation, and the LCR-1214 implementation note provides for continued use of the old terminology “to maintain coherence”.

Reply:
Renaming it the "Prompt database" is fine; I don't see any particular problem, but as John says it will take a while to propagate.

Reply:
how about \emph{Level 1 (Prompt) database} in this case keeping both terms